src/parse_util.rs

```diff
@@ -1120,6 +1120,17 @@ pub fn parse_util_detect_errors_in_ast(
             if jcc.conjunction.has_source() && jcc.job.try_source_range().is_none() {
                 has_unclosed_conjunction = true;
             }
+            let continuations = jcc
+                .parent()
+                .and_then(|job_conj_list| job_conj_list.as_job_conjunction_continuation_list());
+            if let Some(continuations) = continuations {
+                for continuation in continuations.contents() {
```

Comment:
use `continuations` instead of `continuations.contents()`, similarly below. It should behave like a regular slice,
`contents()` should probably be private but no need to do that here.